modelon/impact/client/experiment_definition.py

```diff
@@ -206,6 +218,83 @@ def with_modifiers(self, modifiers=None, **modifiers_kwargs):
             )
         return new
 
+    def with_extensions(self, experiment_extension):
+        """Sets up an experiment with multiple experiment extensions.
+
+        Parameters:
+
+            experiment_extension --
+                "A list of experiment extension objects."
+                "Extension object must an instance of "
+                "modelon.impact.client.experiment_definition."
+                "SimpleExperimentExtension class."
+
+        Example::
+
+            fmu = model.compile().wait()
+            experiment_definition = fmu.new_experiment_definition(custom_function).
+            with_extensions(
+                [
+                    SimpleExperimentExtension().with_modifiers({'PI.k': 20}),
+                    SimpleExperimentExtension().with_modifiers({'PI.k': 30}),
+                ]
+            )
+
+            experiment_definition = fmu.new_experiment_definition(custom_function).
+            with_extensions(
+                [
+                    SimpleExperimentExtension(
+                        parameter_modifiers={'start_time': 0.0, 'final_time': 2.0}
+                    ).with_modifiers({'PI.k': 20})
+                ]
+            )
+        """
+        exp_ext = []
```

Comment:
As it is now, if you do `with_extensions` twice the second will override the first. You could change this line to something like:
`new_extensions = self.extensions + experiment_extensions`
which would fix this.